features/step_definitions/sample_file_steps.rb

```diff
@@ -0,0 +1,219 @@
+Given(/^a demo directory with a smelly file$/) do
+  contents = <<-EOS
+class Dirty
```

Comment:
Ho about using ActiveSupport for this?
-> http://guides.rubyonrails.org/active_support_core_extensions.html#strip-heredoc